spark/src/test/scala/org/apache/spark/sql/delta/deletionvectors/DeletionVectorsSuite.scala

```diff
@@ -259,6 +259,21 @@ class DeletionVectorsSuite extends QueryTest
     }
   }
 
+  test("DELETE with DVs with column mapping mode 'name'") {
+    withTempDir { dirName =>
+      spark.conf.set("spark.databricks.delta.properties.defaults.columnMapping.mode", "name")
+      val path = dirName.getAbsolutePath
+
+      val data = (0 until 50).map(x => (x % 10, x, s"foo${x % 5}"))
+      data.toDF("part", "col1", "col2").write.format("delta").partitionBy(
+        "part").mode("overwrite").save(path)
+      spark.sql(s"alter table delta.`$path`" +
+        s" set tblproperties('delta.enableDeletionVectors' = 'true')")
+      spark.sql(s"DELETE FROM delta.`$path` WHERE col1 = 2")
+      checkAnswer(spark.sql(s"select * from delta.`$path` WHERE col1 = 2"), Seq())
```

Comment:
Please add an assert the table has DVs. I think there are some test utility methods already for this check.

Reply:
now using verifyDVsExist function !